hr_payroll/data/hr_payroll_demo.xml

```diff
@@ -0,0 +1,148 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
Can you post this file in demo directory?
This are the guidelines: https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#directories

Reply:
done